main.cpp

```diff
@@ -1,207 +1,127 @@
-#include <Windows.h>
-#include <MMSystem.h>
-#include <iostream>
-#include <io.h>
-#include "dirent.h"
-#include <vector>
+#include "audioplayer.h"
 
-#pragma comment(lib, "winMM.lib")
+void printMenu();
 
-using namespace std;
-
-void getFileNames(const string &, vector<string> *);
-
-string changeVolume(string currentVolume);
-
-string changeTrack(vector<string>);
+int main() {
+    AudioPlayer musicPlayer;
```

Comment:
Let's rename musicPlayer to audioPlayer for consistency purposes.

Reply:
fixed